models/database.go

```diff
@@ -50,7 +49,7 @@ var databaseSchema = [][]string{
 			custom_labels TEXT,
 			address VARCHAR(255),
 			created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
I would rather not have DEFAULT values in DB – they interact in confusing ways with ORMs, and our `BeforeInsert`/`BeforeUpdate` hooks do it anyway.
That was required for MySQL for a weird reason: https://dev.mysql.com/doc/refman/5.6/en/timestamp-initialization.html